crates/bindings-csharp/BSATN.Runtime/BSATN.Runtime.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <!-- Note: the source code of this package is used in Unity too, which is limited to C# 9 and .NET Standard 2.1. -->
+    <!-- Note: the source code of this package is used in Unity too, which is limited to C# 11 and .NET Standard 2.1. -->
```

Comment:
Good catch, this comment is outdated as of recently. It should be something like this though:

```suggestion
    <!-- Note: the binary produced by this package is used in Unity too, which is limited to .NET Standard 2.1. -->
```